### ISSUE TITLE
set `INSPECT_LOG_DIR` in pkgdown action

### ISSUE THREAD

**Issue by simonpcouch** [simonpcouch/rinspect#82]
The Task help-page shows:

```
#> Warning: ! rinspect could not find a log directory; evaluation log files will be written
#>   to a temporary directory.
#> ℹ Set a log directory with e.g. `rinspect::inspect_log_dir_set("./logs")`,
#>   perhaps in ~/.Rprofile, to quiet this warning.
```
